### PR TITLE
div by zero fixes, detected by address sanitizer

### PR DESCRIPTION
**Pull request by hyperlogic**
Fix for math errors detected by AddressSanitizer on Linux.

### CODE REVIEW

**Comment by hyperlogic** [comment]
QA test plan:
   * smoke test
   * avatars should appear normally.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4025/HighFidelity-Beta-PR11566-cb01c5cada93ed0d4e02f250c7a9047efcc1d227.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4025/HighFidelity-Beta-PR11566-cb01c5cada93ed0d4e02f250c7a9047efcc1d227.exe

File: libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp [hunk 1]
```diff
@@ -446,7 +446,8 @@ void Avatar::applyPositionDelta(const glm::vec3& delta) {
 void Avatar::measureMotionDerivatives(float deltaTime) {
     PerformanceTimer perfTimer("derivatives");
     // linear
-    float invDeltaTime = 1.0f / deltaTime;
+    const float safeDeltaTime = (deltaTime == 0.0f) ? 0.001f : deltaTime;
+    float invDeltaTime = 1.0f / safeDeltaTime;
```
Comment (AndrewMeadows) [inline libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp]: The above logic creates a discontinuity when `deltaTime` reaches `0.0f`.  Maybe should just do:
```
const float MIN_DELTA_TIME = 0.001f;
float invDeltaTime = 1.0f / glm::max(deltatime, MIN_DELTA_TIME);
```

File: interface/src/LODManager.cpp [hunk 2]
```diff
@@ -54,7 +54,8 @@ void LODManager::autoAdjustLOD(float batchTime, float engineRunTime, float delta
     float renderTime = batchTime + OVERLAY_AND_SWAP_TIME_BUDGET;
     float maxTime = glm::max(renderTime, engineRunTime);
     const float BLEND_TIMESCALE = 0.3f; // sec
-    float blend = BLEND_TIMESCALE / deltaTimeSec;
+    const float safeDeltaTime = (deltaTimeSec == 0.0f) ? 0.001f : deltaTimeSec;
+    float blend = BLEND_TIMESCALE / safeDeltaTime;
```
Comment (AndrewMeadows) [inline interface/src/LODManager.cpp]: Similar to the changes in `Avatar::measureMotionDerivatives()` below, this introduces a discontinuity in the blend when `deltaTimeSec` reaches zero.

**Comment by cain-kilgore** [comment]
Testing.

**Comment by cain-kilgore** [comment]
# Test Passed
Test | Result
--- | ---
smoke test | Pass
avatars should appear normally. | Pass

Everything working as expected.

**Comment by themelissabrown** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4831/HighFidelity-Beta-PR11566-cb01c5cada93ed0d4e02f250c7a9047efcc1d227.dmg

File: libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp (hunk 1 above)
Comment (hyperlogic) [reply libraries/avatars-renderer/src/avatars-renderer/Avatar.cpp]: good catch, will fix ASAP

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4843/HighFidelity-Beta-PR11566-ba4c0f189e9600921efa5c4107858775b04b6de3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4843/HighFidelity-Beta-PR11566-ba4c0f189e9600921efa5c4107858775b04b6de3.exe

File: interface/src/LODManager.cpp [hunk 3]
```diff
@@ -54,7 +54,9 @@ void LODManager::autoAdjustLOD(float batchTime, float engineRunTime, float delta
     float renderTime = batchTime + OVERLAY_AND_SWAP_TIME_BUDGET;
     float maxTime = glm::max(renderTime, engineRunTime);
     const float BLEND_TIMESCALE = 0.3f; // sec
-    float blend = BLEND_TIMESCALE / deltaTimeSec;
+    const float MIN_DELTA_TIME = 0.001f;
+    const float safeDeltaTime = glm::max(deltaTimeSec, MIN_DELTA_TIME);
+    float blend = BLEND_TIMESCALE / safeDeltaTime;
```
Comment (AndrewMeadows) [inline interface/src/LODManager.cpp]: Oops, sorry I didn't notice this before.  I think the math here was originally wrong.  It should have been:
```
float blend = deltaTimeSec / BLEND_TIMESCALE;
if (blend > 1.0f) {
    blend = 1.0f;
}
```
This way, when `deltaTimeSec` is very small there is very little blending, but when `deltaTimeSec` is larger than `BLEND_TIMESCALE` it blends immediately.

Comment (AndrewMeadows) [reply interface/src/LODManager.cpp]: Meh, this bug existed before this attempt to fix the div-by-zero problem.  I'll fit it properly in a separate PR and supply a test.

**Pull request merged**